std/algorithm/mutation.d

```diff
@@ -652,10 +652,10 @@ Params:
              representing the _fill pattern.
  */
 void fill(InputRange, ForwardRange)(InputRange range, ForwardRange filler)
-    if (isInputRange!InputRange
-        && (isForwardRange!ForwardRange
-            || (isInputRange!ForwardRange && isInfinite!ForwardRange))
-        && is(typeof(InputRange.init.front = ForwardRange.init.front)))
+if (isInputRange!InputRange
+    && (isForwardRange!ForwardRange
+    || (isInputRange!ForwardRange && isInfinite!ForwardRange))
```

Comment:
Shouldn't this line still be indented? The `||` is part of a sub condition.

```D
(isForwardRange!ForwardRange || (isInputRange!ForwardRange && isInfinite!ForwardRange))
```

Putting it at the same level makes it confusing, with the `&&` and `||` aligned.